yapapi/events.py

```diff
@@ -34,21 +112,128 @@ def exception(self) -> Optional[BaseException]:
 
 @attr.s(auto_attribs=True)
 class JobEvent(Event, abc.ABC):
-    job_id: str
+    job: "Job"
+
+    @property
+    def job_id(self) -> str:
+        return self.job.id
+
+    @property
+    def expires(self) -> datetime:
```

Comment:
For this property I'd stick with the original name `expiration_time` or change the existing one to `expires_at`. The name `expires` is vague and suggests the result could be a `bool`.

Reply:
The "original" name - that is, the name of the `JobEvent` attribute before any changes - was `expires`. I fully agree that this name should change, but this PR is supposed to preserve the attributes. 

I mentioned this in:
https://github.com/golemfactory/yapapi/issues/796

(My particular opinion is that this property should just be removed. What is the purpose? To confuse someone who will think that the event has some expiration date? We have `job`, let's use the `job`)